code/model/DNProject.php

```diff
@@ -509,10 +509,62 @@ public function onAfterDelete() {
 	 * Fetch the public key for this project.
 	 */
 	public function getPublicKey() {
+		$filter = FileNameFilter::create();
```

Comment:
I don't think so. This filters any harmful names (ie ../other-project/ would be other-project).